R/html_document.R

```diff
@@ -49,9 +49,30 @@
 #'  "darkly", "readable", "spacelab", "united", "cosmo", "lumen", "paper",
 #'  "sandstone", "simplex", or "yeti"). Pass \code{NULL} for no theme (in this
 #'  case you can use the \code{css} parameter to add your own styles).
-#'@param highlight Syntax highlighting style. Supported styles include
-#'  "default", "tango", "pygments", "kate", "monochrome", "espresso", "zenburn",
-#'  "haddock", and "textmate". Pass \code{NULL} to prevent syntax highlighting.
+#'@param highlight Syntax highlight engine and style. See the
+#'  \emph{Highlighting} section below for details.
+#'
+#'  "default" (and "textmate") will use highlightjs as syntax highlighting
+#'  engine instead of Pandoc.
+#'
+#'  Any other value will be passed to Pandoc's
+#'  \href{https://pandoc.org/MANUAL.html#syntax-highlighting}{highlighting
+#'  style}. Pandoc's built-in styles include "tango", "pygments", "kate",
+#'  "monochrome", "espresso", "zenburn", "haddock" and "breezedark". Two custom
+#'   are also included, "arrow", for accessible color scheme and "rstudio",
+#'  to replicate RStudio editor theme. A path to a \samp{.theme} file can also be
+#'  passed to be used as a custom highlighting style for Pandoc. Note that custom
+#'  style are available for Pandoc 2.0+.
```

Comment:
```suggestion
#'   are also included: "arrow", an accessible color scheme, and "rstudio",
#'   which mimics the default IDE theme. 
#' 
#'  Alternatively, supply a path to a \samp{.theme} to use a custom Pandoc style.
#'  (This requires Pandoc 2.0+)

Reply:
I regrouped in one paragraph as `arrow` and `rstudio` also require pandoc 2.0+